HandheldCompanion/Views/QuickPages/QuickDevicePage.xaml.cs

```diff
@@ -21,6 +23,7 @@ public QuickDevicePage()
     {
         InitializeComponent();
 
+        FullFanSpeedToggler.Visibility = MainWindow.CurrentDevice is LegionGo ? Visibility.Visible : Visibility.Hidden;
```

Comment:
@MSeys The check to hide if for non LeGo devices is actually here. Should I move it?